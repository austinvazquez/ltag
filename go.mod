module github.com/kunalkushwaha/ltag

go 1.17
