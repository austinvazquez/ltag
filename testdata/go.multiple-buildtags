//go:build (amd64 || arm64) && !static_build && !gccgo
// +build amd64 arm64
// +build !static_build
// +build !gccgo

// Package plugin does something.
package plugin
