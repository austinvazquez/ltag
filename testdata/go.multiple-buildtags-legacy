// +build amd64 arm64
// +build !static_build
// +build !gccgo

// Package plugin does something.
package plugin
