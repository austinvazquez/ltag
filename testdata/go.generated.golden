// Code generated by protoc-gen-go. DO NOT EDIT.
// versions:
// 	protoc-gen-go v1.28.1
// 	protoc        v3.20.1
// source: github.com/containerd/containerd/api/types/metrics.proto

package types
